javascript/ql/src/semmle/javascript/frameworks/SystemCommandExecutors.qll

```diff
@@ -51,13 +56,19 @@ private class SystemCommandExecutors extends SystemCommandExecution, DataFlow::I
         )
         or
         shell = true and
-        mod = "exec" and
-        optionsArg = -2 and
-        cmdArg = 0
+        (
+          mod = "exec" and
+          optionsArg = -2 and
+          cmdArg = 0
+          or
+          mod = "remote-exec" and
```

Comment:
Did you mean to sneak in support for `remote-exec` here? Which is fine of course, just checking.

Reply:
Apparently I managed to duplicate that (it's already defined further down). Will fix.